### ISSUE TITLE
[bug] conan list --graph with non-existing file

### ISSUE THREAD

**Issue by Todiq** [conan-io/conan#16290]
### Describe the bug

version: 2.3.1
conan_path: D:\venv\Scripts\conan
python
  version: 3.9.13
  sys_version: 3.9.13 (tags/v3.9.13:6de2ca5, May 17 2022, 16:36:42) [MSC v.1929 64 bit (AMD64)]
  sys_executable: D:\venv\Scripts\python.exe
  is_frozen: False
  architecture: AMD64
system
  version: 10.0.22631
  platform: Windows-10-10.0.22631-SP0
  system: Windows
  release: 10
  cpu: Intel64 Family 6 Model 186 Stepping 3, GenuineIntel

### How to reproduce it

Simply run `conan list --graph nonexistingfile.json`. It gives:

```shell
ERROR: Traceback (most recent call last):
  File "D:\venv\lib\site-packages\conan\cli\cli.py", line 291, in main
    cli.run(args)
  File "D:\venv\lib\site-packages\conan\cli\cli.py", line 193, in run
    command.run(self._conan_api, args[0][1:])
  File "D:\venv\lib\site-packages\conan\cli\command.py", line 164, in run
    info = self._method(conan_api, parser, *args)
  File "D:\venv\lib\site-packages\conan\cli\commands\list.py", line 250, in list
    pkglist = MultiPackagesList.load_graph(graphfile, args.graph_recipes, args.graph_binaries)
  File "D:\venv\lib\site-packages\conan\api\model.py", line 93, in load_graph
    graph = json.loads(load(graphfile))
  File "D:\venv\lib\site-packages\conans\util\files.py", line 133, in load
    with open(path, 'r', encoding=encoding, newline="") as handle:
FileNotFoundError: [Errno 2] No such file or directory: 'D:\\nonexistingfile.json'

ERROR: [Errno 2] No such file or directory: 'D:\\nonexistingfile.json'
```

Same behaviour on Linux.

**Comment by AbrilRBS** [comment]
Thanks for the report, we'll make it have a prettier error in these cases

**Comment by memsharded** [comment]
https://github.com/conan-io/conan/pull/16291 merged for next Conan 2.4
